app/views/cooks/_form.html.erb

```diff
@@ -0,0 +1,19 @@
+
+  <div class="field">
+    <%= form.label :Cook %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= form.label :Cuisine %>
+    <%= form.text_field :title %>
+  </div>
+
+  <div class="field">
+    <%= form.label :Details %>
+    <%= form.text_area :content %>
+  </div>
+
+  <div class="actions">
+    <%= form.submit %>
+  </div>
```

Comment:
Is this file used anywhere?

Reply:
Nice find! I've deleted this now, thanks!